task-maker-format/src/ioi/sanity_checks/att.rs

```diff
@@ -128,6 +128,7 @@ impl SanityCheck<IOITask> for AttSampleFilesValid {
                     eval,
                     format!("Validation of sample case {}", input_name.display()),
                     0,
+                    None,
```

Comment:
I'm wondering if this should be the name of the subtask with the examples. Not sure how to reliably get that, or whether a new subtask name should be used (i.e. "att"). Probably leaving that empty is fine for now.